### PR TITLE
fix(name_profile): SPK-55397 Added "Saint ###" county name variant

### PR DESCRIPTION
**Pull request by jperna7254**
Anywhere that a county name was spelled with "ST." I added a duplicate with the spelling "SAINT".

### CODE REVIEW

**Review by vlam-spokeo — APPROVED** [review]

**Comment by fhuspokeo** [comment]
probably being a little nit-picky, but could you put them in alphabetical order?

**Review by fhuspokeo — APPROVED** [review]

File: lib/data/COUNTY_NAME_TO_CODE.yml [hunk 1]
```diff
@@ -194,6 +195,7 @@
     WHITE: 145
     WOODRUFF: 147
     YELL: 149
+    SAINT FRANCIS: 123
```
Comment (mohitgpt24) [inline lib/data/COUNTY_NAME_TO_CODE.yml:198]: isn't it same as line 190?

**Comment by mohitgpt24** [comment]
I am referencing https://www2.census.gov/programs-surveys/decennial/2010/partners/pdf/FIPS_StateCounty_Code.pdf, and I don't see `Saint ..`, but its `St.`. Is it because our data needs the mapping for "Saint .."?

**Comment by jperna7254** [comment]
@mohitgpt24 The Name Presentation data seems to have the "Saint" spelling so I am including that variant.

**Pull request merged**